<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1546857071941' stop='1546857106399'>
    <name>Login as Hiring Manager</name>
    <title>Login as Hiring Manager</title>
    <test-cases>
        <test-case start='1546857089323' status='failed' stop='1546857106398'>
            <name>Raise Requisition Request</name>
            <title>Raise Requisition Request</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='93dad355-8012-402f-8855-d8511479c49f-attachment.png' type='image/png' size='77279'/>
            </attachments>
            <failure>
                <message>Failed: unknown error: Element &lt;span _ngcontent-c18="" class="choosefilespan">...&lt;/span> is not clickable at point (703, 470). Other element would receive the click: &lt;input _ngcontent-c18="" id="avatar" multiple="" name="avatar" type="file">
                  (Session info: chrome=71.0.3578.98)
                  (Driver info: chromedriver=2.42.591071 (0b695ff80972cc1a65a5cd643186d2ae582cd4ac),platform=Linux 4.15.0-43-generic x86_64)</message>
                <stack-trace>WebDriverError: unknown error: Element &lt;span _ngcontent-c18="" class="choosefilespan">...&lt;/span> is not clickable at point (703, 470). Other element would receive the click: &lt;input _ngcontent-c18="" id="avatar" multiple="" name="avatar" type="file">
                  (Session info: chrome=71.0.3578.98)
                  (Driver info: chromedriver=2.42.591071 (0b695ff80972cc1a65a5cd643186d2ae582cd4ac),platform=Linux 4.15.0-43-generic x86_64)
                    at Object.checkLegacyResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:546:15)
                    at parseHttpResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:509:13)
                    at doSend.then.response (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: WebElement.click()
                    at Driver.schedule (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
                    at WebElement.schedule_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
                    at WebElement.click (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2092:17)
                    at actionFn (/usr/local/lib/node_modules/protractor/built/element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at actionResults.getWebElements.then (/usr/local/lib/node_modules/protractor/built/element.js:461:65)
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)Error
                    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
                    at Object.clickWhenClickable (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/node_modules/protractor-helper/index.js:90:15)
                    at UserContext.&lt;anonymous> (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/Testcases/raiserequisitionrequest.js:42:26)
                    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
                    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
                    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
                    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                From: Task: Run it("Raise Requisition Request") in control flow
                    at UserContext.&lt;anonymous> (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at Spec.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
                    at UserContext.fn (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/Testcases/raiserequisitionrequest.js:11:5)
                    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/home/cnt-dev/Documents/Chutti-RMT/requisition-management/Testcases/raiserequisitionrequest.js:8:1)
                    at Module._compile (internal/modules/cjs/loader.js:689:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
                    at Module.load (internal/modules/cjs/loader.js:599:32)
                    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>